src/Card/Card.stories.js

```diff
@@ -288,9 +274,7 @@ const CardTemplateMinimalRight = ({ ...rest }) => (
   <Card {...rest}>
     <Card.Image image={{ filename: "https://placekitten.com/600/300" }} />
     <Card.Content align="right" className="su-rs-pt-2">
-      <Card.Superhead className="su-font-sans su-font-bold su-type-0 su-mb-0">
-        Super headline
-      </Card.Superhead>
+      <Card.Superhead weight="regular">Super headline</Card.Superhead>
```

Comment:
@rebeccahongsf you could safely take the classnames/props out if they're the same as default, and just add the props/classnames that are different for this example 😃